### DISCUSSION TITLE
Can I deploy perfect server without docker ?

### DISCUSSION THREAD

**Discussion by fanofthis** [PrefectHQ/prefect#4681] [Q&A]
I want to deploy prefect server without using docker,
because my OS server is windows server 2012 and not support docker.

**Comment by pikhovkin** [comment]
linked with #4370

**Comment by zanieb** [comment]
We cannot explicitly support this. Especially on Windows, this will be difficult.

See previous discussion at 
https://github.com/PrefectHQ/server/issues/79
https://github.com/PrefectHQ/prefect/discussions/3731
